lib/request.ex

```diff
@@ -26,7 +27,7 @@ defmodule ApolloIo.Request do
         {:ok, body, headers}
 
       {:ok, %Req.Response{body: body, status: _error}} ->
-        {:error, body}
+        {:error, %ApolloIo.Error{message: body}}
```

Comment:
@neilberkman I tried to match the error response for the both `GET` & `POST` same to be consistent. Let me know if you have any concerns on this.